src/popper/runner_kubernetes.py

```diff
@@ -263,16 +263,22 @@ def _init_pod_delete(self):
     def _vol_create(self, volume_name):
         """Create a default PersistentVolume of hostPath type.
         """
+        hostpathvol_path = "/tmp"
+        hostpathvol_size = "1Gi"
+        if self._config.resman_opts.get("hostpathvol_path", None):
+            hospathvol_path = self._config.resman_opts.hostpathvol_path
```

Comment:
There is a typo in the variable name. It is `hospathvol_path`. Can you please fix this ?

Reply:
To catch typos like this, it will be a good idea to test it locally on a minikube cluster maybe. If its too much work, its fine, I can do it on my end.

Reply:
Thanks! Good to know. I'll look into that.